src/components/TestDateCard/index.tsx

```diff
@@ -19,8 +27,41 @@ const TestDateCard = ({
   testIndex,
   score,
   requestId,
+  requestName,
   id
 }: TestDateCardProps) => {
+  const { t } = useTranslation('accessibility');
+  const history = useHistory();
+
+  // TODO: should we do something w/ mutationResult here?
+  const [mutate] = useMutation<RemoveTestDate>(RemoveTestDateQuery, {
+    errorPolicy: 'all'
+  });
+
+  const [isRemoveTestDateModalOpen, setRemoveTestDateModalOpen] = useState(
+    false
+  );
+
+  const removeTestDate = () => {
+    mutate({
+      variables: {
+        input: {
+          id
+        }
+      }
+    }).then(() => {
+      history.push(`/508/requests/${requestId}`, {
+        confirmationText: t('removeTestDate.confirmation', {
```

Comment:
I wonder if we could call `setConfirmationText` directly? I know you and Isaac are going to chat about this. Let me know if you need more eyes.